Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Levi9.ERP.Domain.Models.DTO
+{
+    public class DocumentItemDTO
+    {
+        public int ProductId { get; set; }
+        public string Name { get; set; }
+        public float PriceValue { get; set; }
+        public string Currency { get; set; }
```

Comment:
use enum for Currency

Reply:
Resolved

Reply:
Cannot find enum CurrencyType in DocumentItemDTO